### PR TITLE
Use submit_to instead of individual config params

### PR DESCRIPTION
**Pull request by ancorgs**
No changelog entry or version bump, but let's do it after the branching to avoid side effects, needing a bugzilla number and so on

### CODE REVIEW

File: ytools/yast2/create_maintenance_branch [hunk 1]
```diff
@@ -8,21 +8,18 @@
...
-DEVEL_PROJECT="YaST:openSUSE:13.2"
-TARGET_PROJECT="openSUSE:13.2:Update"
-BUILD_TARGET="openSUSE_13.2"
-
+#
+BRANCH_NAME="openSUSE-42_1"
+# See https://github.com/yast/yast-rake/blob/master/data/targets.yml
+TARGET="leap_42_1"
+# Individual config options are not usually needed now that we have submit_to
+CONF_OPTIONS = {}
```
Comment (jreidinger) [inline ytools/yast2/create_maintenance_branch]: why leap as default?

File: ytools/yast2/create_maintenance_branch [hunk 2]
```diff
@@ -45,24 +42,36 @@ def modify_rakefile
...
-    lines.insert(line_index, conf_line, "end\n")
+    lines.insert(2, new_line, "\n")
   end
 
-  CONF_OPTIONS.each do |key, value|
-    config_line = lines.grep(/#{conf_var}\.#{key}\s*=/).first
-    new_line = "  #{conf_var}.#{key} = #{value.inspect}\n"
-    if config_line
-      lines[lines.index(config_line)] = new_line
+  if !CONF_OPTIONS.empty?
```
Comment (jreidinger) [inline ytools/yast2/create_maintenance_branch]: lets drop it. I think for yast we use new way. Old one is only for manual testing. No need to use it during creation of maintenance branch.

File: ytools/yast2/create_maintenance_branch (hunk 1 above)
Comment (ancorgs) [inline ytools/yast2/create_maintenance_branch]: there is no target for openSUSE-13_2 in the rake tasks

File: ytools/yast2/create_maintenance_branch (hunk 2 above)
Comment (ancorgs) [inline ytools/yast2/create_maintenance_branch]: Well, if we drop this maybe it also makes sense to change the hacky constants based approach and simply provide BRANCH_NAME and TARGET in the command line.

Comment (jreidinger) [inline ytools/yast2/create_maintenance_branch]: yes it makes sense

File: ytools/yast2/create_maintenance_branch (hunk 1 above)
Comment (jreidinger) [inline ytools/yast2/create_maintenance_branch]: so maybe use sle12 sp1 as default?

**Comment by jreidinger** [comment]
LGTM

**Pull request merged**